packages/admin/admin/src/mui/menu/Menu.styles.ts

```diff
@@ -1,58 +1,66 @@
-import { Theme } from "@mui/material";
-import { createStyles } from "@mui/styles";
+import { CSSObject, Drawer as MuiDrawer, DrawerProps, Theme } from "@mui/material";
+import { styled } from "@mui/material/styles";
+import { createStyles, StyledComponent } from "@mui/styles";
+import { MUIStyledCommonProps } from "@mui/system";
+import * as React from "react";
 
-import { MenuProps } from "./Menu";
+import { MasterLayoutContext } from "../MasterLayoutContext";
+import { DEFAULT_DRAWER_WIDTH, DEFAULT_DRAWER_WIDTH_COLLAPSED, MenuProps } from "./Menu";
 
-export type MenuClassKey = "drawer" | "permanent" | "temporary" | "open" | "closed";
+const openedMixin = (theme: Theme, drawerWidth?: number): CSSObject => ({
+    width: drawerWidth ?? DEFAULT_DRAWER_WIDTH,
+    transition: theme.transitions.create("width", {
+        easing: theme.transitions.easing.sharp,
+        duration: theme.transitions.duration.enteringScreen,
+    }),
+    overflowX: "hidden",
```

Comment:
Since this is set, both when opening and when closing, can this just be added to the normal styles instead?

Reply:
@jomunker will you change this, or is there a reason for doing it this was?

Reply:
It's also done in the follow up PR